common/api/summary/tree-widget-react.exports.csv

```diff
@@ -1,19 +1,19 @@
 sep=;
 Release Tag;API Item
 public;areAllModelsVisible(models: string[], viewport: Viewport): boolean
-public;BaseFilterableTreeProps
-public;BaseTreeProps
+public;BaseFilterableTreeProps 
+public;BaseTreeProps 
 public;CategoriesTreeComponent:
-public;CategoriesTreeComponentProps
-public;CategoriesTreeHeaderButtonProps
+public;CategoriesTreeComponentProps 
```

Comment:
Not sure. I noticed that after running `extract-api` and opening `export` or `api` file vs code does some formatting on save. 😕

Reply:
I also have "format on save" enabled in my VSCode, but it formats all file, not just some of it 🤔

Reply:
Looks like saving it again fixes. Same for `api` file. I guess we should commit `extract-api` produced files without additional formatting from vs code?

Reply:
> I guess we should commit `extract-api` produced files without additional formatting from vs code?

yes, otherwise CI will complain that the API extractions weren't generated

Reply:
Pushed unmodified extract-api output.